modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/utils/fieldSupport.js

```diff
@@ -468,21 +465,22 @@ export const createField = (props, event) => {
 	};
 };
 
-export const formatEditorConfig = (editorConfig, instanceId) => {
-	Object.keys(editorConfig).map((key) => {
-		if (typeof editorConfig[key] === 'string') {
-			const parsedName = parseName(decodeURIComponent(editorConfig[key]));
+export const updateEditorConfigInstanceId = (editorConfig, instanceId) => {
+	const newEditorConfig = {...editorConfig};
```

Comment:
I would name this constant as `updatedEditorConfig`, since we are instantiating an object using the properties of an existing one:
```suggestion
	const updatedEditorConfig = {...editorConfig};
```